data/css/mesh.scss

```diff
@@ -49,6 +49,8 @@ $display-font-composite: 'Roboto' !default;
 
 @import 'modules/window/simple';
 
+@import 'widgets/tabButton';
```

Comment:
Because that would mean that the tabButton styling is contigent on whether we are using the hamburgerBasement template somewhere in the app. If we decided to change the template on the front page to something other than hamburgerBasement, we would then lose the styling for the tab button on the section page. I'd rather they were separate concerns